web/impact/impact/permissions/graphql_permissions.py

```diff
@@ -0,0 +1,43 @@
+from accelerator.models import (
+    UserRole,
+)
+from accelerator_abstract.models.base_user_utils import is_employee
+from accelerator.models import ACTIVE_PROGRAM_STATUS
+
+
+def visible_roles(current_user):
+    basic_user_roles = [
+        UserRole.FINALIST,
+        UserRole.AIR,
+        UserRole.MENTOR,
+        UserRole.PARTNER,
+        UserRole.ALUM
+    ]
+    basic_visible_roles = [UserRole.FINALIST, UserRole.STAFF, UserRole.ALUM]
+
+    current_logged_in_user_roles = list(
+        current_user.programrolegrant_set.filter(
+            program_role__program__program_status=ACTIVE_PROGRAM_STATUS
+        ).values_list(
+            'program_role__user_role__name', flat=True).distinct()
+    )
+    if not current_logged_in_user_roles:
+        return [UserRole.STAFF]
+    if set(basic_user_roles).intersection(current_logged_in_user_roles):
```

Comment:
In future, consider using a construction like

```
if any([role in basic_user_roles for role in current_logged_in_user_roles]):
   ....
```
instead of set intersections, to help readability. This will allow the reader to focus more directly on the logic.